changelog/front-commerce-3.2-2.28.mdx

```diff
@@ -0,0 +1,253 @@
+---
+title: "Release: 3.2 & 2.28"
+authors: pierre
+date: 2024-01-08T10:00
+hide_table_of_contents: true
+---
+
+## New packages, Magic Button revamp, Attraqt recommendations and Cloud improvements!
+
+Happy new year everyone! We hope you had a great holiday season and are ready to
+start the new year with a bang.
+
+We are excited to introduce a range of new features and enhancements to
+Front-Commerce's core. While we still prioritize core fixes and stabilization,
+**we have been able to ship some significant features to Front-Commerce
+Remixed**: all Magento1 modules, Adyen payment and Akamai Image Manager are now
+available as standalone packages under the `@front-commerce/***` namespace.
+
+We have also **revamped the technical foundations of the Magic Button** and
+introduced a new concept called **server Services** in Front-Commerce 3.2.
+
+The `2.x` branch also received some love with the introduction of **Attraqt
+personalized recommendations** and a few bug fixes. **Be sure to upgrade to
+Front-Commerce 2.28!** As time progresses, we anticipate more projects
+transitioning to `3.x`, and the number of new features released for both major
+versions will gradually decrease.
+
+Let's dive in and explore what's new in Front-Commerce, and for Front-Commerce
+Cloud customers…
+
+<!-- truncate -->
+
+import ChangelogFooter from "@site/src/components/ChangelogFooter";
+import BackportList from "@site/src/components/BackportList";
+import ContactLink from "@site/src/components/ContactLink";
```

Comment:
This is no longer needed, since this commit: https://github.com/front-commerce/developers.front-commerce.com/commit/5cde2a7bb491dfe528026b2d2b99c335bf0153d4

Reply:
For more information see: https://docusaurus.io/docs/markdown-features/react#mdx-component-scope

Reply:
Indeed. Copy-paste!

Thanks.